modules/core/src/main/java/org/apache/synapse/message/processor/impl/forwarder/ForwardingService.java

```diff
@@ -214,7 +237,7 @@ public void execute() {
 						// massages.
 						if (log.isDebugEnabled()) {
 							log.debug("No messages were received for message processor [" +
-							          messageProcessor.getName() + "]");
+									messageProcessor.getName() + "]");
```

Comment:
Operator should precede the line shall we fix this?

Reply:
Fixed. Will look into other places as well.